controller/src/main/java/uni/miskolc/ips/ilona/navigation/controller/WayfindingController.java

```diff
@@ -1,65 +1,62 @@
 package uni.miskolc.ips.ilona.navigation.controller;
 
-import java.util.ArrayList;
-import java.util.List;
-
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.ResponseBody;
-
 import uni.miskolc.ips.ilona.measurement.model.position.Zone;
 import uni.miskolc.ips.ilona.navigation.controller.dto.NavigationPersonRequest;
 import uni.miskolc.ips.ilona.navigation.controller.dto.NavigationZoneRequest;
 import uni.miskolc.ips.ilona.navigation.service.NoRouteAvailableException;
 import uni.miskolc.ips.ilona.navigation.service.WayfindingService;
 
+import java.util.ArrayList;
+import java.util.List;
+
 
 @Controller
 public class WayfindingController {
-	
-	private WayfindingService wayfindingService;
+    @Autowired
```

Comment:
Use Autowired on Constrcutor of setter method instead of field directly. It facilitates testing.